sketch_blink.ino.ino

```diff
@@ -0,0 +1,44 @@
+
+/*this fade sketch meets the Acceptance Criteria https://trello.com/c/HQigFpAG */
+
+int pin = 9;
+int shine = 0;
+int fadeAmount = 5.9;
+int flashon = 258;
+int flashoff = 250;
+
```

Comment:
You could use the camel case for flashon and flashoff (flashOn, flashOff) to be consistent with the other variable names and to follow good practice for code writing.